feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/TimetableItem.kt

```diff
@@ -33,26 +37,33 @@ fun TimetableItem(
     } else {
         Color(roomColor).copy(alpha = 0.2F)
     }
-    Column(
-        modifier
-            .background(color, MaterialTheme.shapes.medium)
-            .border(2.dp, Color(roomColor), MaterialTheme.shapes.medium)
-            .padding(8.dp)
-            .semantics { contentDescription = "isFavorited$isFavorited" }
-            .testTag("favorite")
+    val localDensity = LocalDensity.current.let {
+        Density(it.density * verticalScale, it.fontScale)
+    }
```

Comment:
Just in my opinion...
Maybe, the text size does not need to be scaled for visibility.
For example, only the height of the session items is scaled down, and overflowing text is displayed as "...".
However, this is not what the original issue was looking for, so I think it is fine as is. 👍